src/site/content/en/blog/preload-optional-fonts/index.md

```diff
@@ -69,15 +69,15 @@ font. If the font is not downloaded within 100ms, then the fallback font is used
 occurs.
 
 <figure class="w-figure">
-  <img src="./previous-behavior-fail.png" alt="Diagram showing previous optional font behavior when font fails to load">
-  <figcaption class="w-figcaption">Previous <code>font-display: optional</code> behavior in Chrome when font is downloaded <b>after</b> the 100ms block period</figcaption>
+  {% Img src="image/admin/WHLORYEu864QRRveFQUz.png", alt="Diagram showing previous optional font behavior when font fails to load", width="800", height="340" %}
+  <figcaption class="w-figcaption">>Previous <code>font-display: optional</code> behavior in Chrome when font is downloaded <b>after</b> the 100ms block period</figcaption>
```

Comment:
It looks like an extra greater than sign was added:

`<figcaption class="w-figcaption">>`

It looks like this happens more than once in this file so be sure to check for it across all of the changes.